cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/CloudControllerRestClientImpl.java

```diff
@@ -2335,6 +2334,11 @@ private String getNextUrlValue(Map<String, Object> map, String nextUrlKey) {
         return (String) map.get(nextUrlKey);
     }
 
+    private String getNextUrlValueV3(Map<String, Object> map, String nextUrlKey) {
```

Comment:
Remove the useless argument `nextUrlKey`.

Reply:
Done